adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala

```diff
@@ -295,16 +291,19 @@ sealed trait AlignmentRecordRDD extends AvroReadGroupGenomicRDD[AlignmentRecord,
    * Saves an RDD of ADAM read data into the SAM/BAM format.
    *
    * @param filePath Path to save files to.
-   * @param asSam Selects whether to save as SAM or BAM. The default value is true (save in SAM format).
+   * @param asType Selects whether to save as SAM, BAM, or CRAM. The default
+   *   value is None, which means the file type is inferred from the extension.
    * @param asSingleFile If true, saves output as a single file.
    * @param isSorted If the output is sorted, this will modify the header.
    */
   def saveAsSam(
     filePath: String,
-    asSam: Boolean = true,
+    asType: Option[SAMFormat] = None,
```

Comment:
Nah, alas, the enum is just SAM, BAM, and CRAM: https://github.com/HadoopGenomics/Hadoop-BAM/blob/master/src/main/java/org/seqdoop/hadoop_bam/SAMFormat.java#L32